obs_img_utils/utils.py

```diff
@@ -511,3 +511,40 @@ def filter_packages_by_name(packages_metadata, package_name):
         matching_packages[name] = packages_metadata[name]
 
     return matching_packages
+
+
+def get_condition_list_from_file(filename, logger):
+    """
+    Creates the conditions objects from a file with json dump.
+    """
+    with open(filename, 'r') as file:
+        data = json.load(file)
```

Comment:
explodes on syntax error in the file. Needs a try-catch block to gracefully handle syntax errors.